Doc/library/datetime.rst

```diff
@@ -1146,44 +1294,48 @@ Instance methods:
 .. method:: datetime.timetuple()
 
    Return a :class:`time.struct_time` such as returned by :func:`time.localtime`.
-   ``d.timetuple()`` is equivalent to ``time.struct_time((d.year, d.month, d.day,
-   d.hour, d.minute, d.second, d.weekday(), yday, dst))``, where ``yday =
-   d.toordinal() - date(d.year, 1, 1).toordinal() + 1`` is the day number within
-   the current year starting with ``1`` for January 1st. The :attr:`tm_isdst` flag
-   of the result is set according to the :meth:`dst` method: :attr:`.tzinfo` is
-   ``None`` or :meth:`dst` returns ``None``, :attr:`tm_isdst` is set to ``-1``;
-   else if :meth:`dst` returns a non-zero value, :attr:`tm_isdst` is set to ``1``;
-   else :attr:`tm_isdst` is set to ``0``.
+
+   ``d.timetuple()`` is equivalent to::
+
+     time.struct_time((d.year, d.month, d.day, d.hour, d.minute, d.second, d.weekday(), yday, dst))
```

Comment:
This creates horizontal scrolling (at least for me), can the line be broken up, like:

```
    time.struct_time((d.year, d.month, d.day,
                      d.hour, d.minute, d.second,
                      d.weekday(), yday, dst))
```